index.html

```diff
@@ -1222,6 +1222,25 @@ <h3>
             Updating the manifest
           </h3>
           <aside class="issue" data-number="446"></aside>
+          <p>
+            The manifest is fetched and processed on every page load. When the
+            processing is successful, user agents MAY apply updated manifest to
+            any current and future <a>application contexts</a>
+            associated with the application. 
+          </p>
+          <p>
+            User agents SHOULD be careful about performing update to manifest
+            fields like app name and icons. Users should be fully aware if
+            such fields are to be be updated to avoid web applications being
+            installed and silently changed to appear as a different one.
+          </p>
+          <p>
+            When the user agent fetches a manifest as part of an update check,
```

Comment:
I think we should  move this section to the "processing a manifest" section as part of the algorithm: i.e. something like: 
 
1. If the document "processed manifest" is not null, and document [=processed manifest=]'s id member [=URL/equals=] manifest's id member, return. 

(i.e., prevent overriding an existing "processed manifest").

Reply:
I just realized that [=URL/equals=] allows us to "exclude fragments" as an argument, which might mean we can keep the fragment in the `id`, but ignore it for these comparisons. That might simplify `id` a little bit, because we don't need to chop the fragments anymore.